src/app/components/SubHeading/index.stories.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import SubHeading from './index';
+
+const props = {
+  blocks: [
+    {
+      model: {
+        blocks: [
+          {
+            model: {
+              text: 'Testing the SubHeading!',
```

Comment:
This is the text that's displayed within Storybook - so if it can remove the 'testing' part, that'd be super.

Reply:
done